packages/web3-signature/package.json

```diff
@@ -43,7 +43,7 @@
   "dependencies": {
     "@requestnetwork/types": "0.27.0",
     "@requestnetwork/utils": "0.26.0",
-    "web3-eth": "1.3.0"
+    "ethers": "4.0.48"
```

Comment:
Why not use the latest version?
Ethers 5 is out of beta and is the stable ethers version now.

Reply:
Because We use the 4.0.48 version in the rest of the code. I followed a @benjlevesque 's suggestion.

If we want to update to the 5. We should do it for all the repo.